src/transactions/transactions.util.ts

```diff
@@ -53,10 +69,32 @@ export async function processTransaction<
 >(
   method: Method<MethodArgs, ReturnType, TransformedReturnType>,
   args: MethodArgs,
-  opts: ProcedureOpts
+  opts: ProcedureOpts,
+  dryRun = false
 ): Promise<TransactionResult<TransformedReturnType>> {
   try {
     const procedure = await prepareProcedure(method, args, opts);
+    const {
+      fees,
+      payingAccountData: { balance, type, account },
+    } = await procedure.getTotalFees();
+    const supportsSubsidy = await procedure.supportsSubsidy();
```

Comment:
```suggestion
    const promises = [ procedure.getTotalFees(), procedure.supportsSubsidy()]
    const [{
      fees,
      payingAccountData: { balance, type, account },
    },
    supportsSubsidy,
    ] = await Promise.all(promises)
```

Parallelize network call whenever possible.

It's a little less readable, but should improve the time for every transaction, especially when the REST API and chain node aren't colocated.